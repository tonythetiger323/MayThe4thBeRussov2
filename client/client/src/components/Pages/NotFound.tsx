import * as React from 'react'
import Page from '../Layouts/Page'
import Typography from '@material-ui/core/Typography'

const NotFound = (): JSX.Element => {
  return (
    <Page>
      <Typography variant='h1'>Page Not Found</Typography>
    </Page>
  )
}

export default NotFound
