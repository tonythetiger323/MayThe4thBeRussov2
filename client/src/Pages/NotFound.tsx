import * as React from 'react'
import Typography from '@material-ui/core/Typography'

const NotFound = (): JSX.Element => {
  return <Typography variant='h1'>Page Not Found</Typography>
}

export default NotFound
