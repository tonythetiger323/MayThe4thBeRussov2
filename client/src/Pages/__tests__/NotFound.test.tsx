import * as React from 'react'
import { shallow } from 'enzyme'
import NotFound from '../NotFound'

describe('NotFound Page', () => {
  const notFound = shallow(<NotFound />)

  it('should render correctly', () => {
    expect(notFound).toMatchSnapshot()
  })
})
