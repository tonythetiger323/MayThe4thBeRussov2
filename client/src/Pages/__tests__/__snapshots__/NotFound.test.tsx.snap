// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound Page should render correctly 1`] = `
<WithStyles(Typography)
  variant="h1"
>
  Page Not Found
</WithStyles(Typography)>
`;
